src/common/rbac/project/visitor_role.go

```diff
@@ -100,6 +100,13 @@ var (
 			{Resource: rbac.ResourceRobot, Action: rbac.ActionUpdate},
 			{Resource: rbac.ResourceRobot, Action: rbac.ActionDelete},
 			{Resource: rbac.ResourceRobot, Action: rbac.ActionList},
+
```

Comment:
Is the file mask change `644` -> `755` intentional?

Reply:
> Is the file mask change `644` -> `755` intentional?

yep, but I don't know what happened.   reason to be found.